tests/plan/create-kms-key/versions.tf

```diff
@@ -11,3 +11,9 @@ terraform {
 provider "aws" {
   region = var.region
 }
+
+
+provider "aws" {
+  region = var.domino_cur.region
```

Comment:
Given that this will only work in `us-east-1` there is no point in having it as a variable. I would also update the alias to something like `us-east-1`